src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs

```diff
@@ -36,5 +39,16 @@ public string Title
 				WebAssemblyRuntime.InvokeJS(command);
 			}
 		}
+
+		public bool TryEnterFullScreenMode() => SetFullScreenMode(true);
+
+		public void ExitFullScreenMode() => SetFullScreenMode(false);
+
+		private bool SetFullScreenMode(bool turnOn)
+		{
+			var jsEval = $"{ApplicationViewTsType}.setFullScreenMode({turnOn.ToString().ToLowerInvariant()})";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4878148)

Reply:
Fixed